pandas/tests/io/test_parquet.py

```diff
@@ -504,6 +504,7 @@ def test_empty_dataframe(self, pa):
         df = pd.DataFrame()
         check_round_trip(df, pa)
 
+    @pytest.mark.skip(reason="broken test")
```

Comment:
shouldn't this be an xfail?

Reply:
It's a test that got broken recently, because we were not testing it .. (https://github.com/pandas-dev/pandas/issues/29976). Will fix it one of the coming days, but so for now needs to be skipped (or xfailed, that's semantically more correct yes)